src/smt/smt_engine.cpp

```diff
@@ -4035,7 +4018,7 @@ void SmtEnginePrivate::processAssertions() {
   if ( options::bvAbstraction() &&
       !options::incrementalSolving()) {
     dumpAssertions("pre-bv-abstraction", d_assertions);
```

Comment:
The PreprocessingPass class automatically adds support for dumping before and after the pass, so those are not needed.